src/Microsoft.OData.Core/ODataNotificationStream.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.OData
     /// <summary>
     /// Wrapper to listen for dispose on a <see cref="Stream"/>.
     /// </summary>
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NETCOREAPP

```

Comment:
Why NETCOREAPP?

Reply:
Here I was just trying to see if this would work on CI since I was getting tests failing on CI when using `NETCOREAPP3_1_OR_GREATER`. But this didn't help. I have returned it back to `NETCOREAPP3_1_OR_GREATER` and manually defined the constant in the csproj file.